### ISSUE TITLE
Could Avalonia supports SkiaSharp 3.0.0-preview2.1?

### ISSUE THREAD

**Issue by SignalMQ** [AvaloniaUI/Avalonia#14886]
### Describe the bug

**Hi everyone!**
After updating all nuget packages in my project, due to the new release of `Avalonia version 11.0.10`, the project stopped running. I discovered that this is related to `SkiaSharp`, since I have it installed as a separate nuget package version `3.0.0-preview2.1`. Perhaps Avalonia stopped launching due to the fact that something changed in the SkiaSharp architecture.

### To Reproduce

1) Create some AvaloniaMVVM application with `Avalonia 11.0.10`
2) Install to project `SkiaSharp 3.0.0-preview2.1` package
3) Run project

### Avalonia version

11.0.10

### OS

Linux

### Additional context

```
Unhandled exception. System.TypeInitializationException: The type initializer for 'SkiaSharp.SKFontManager' threw an exception.
 ---> System.TypeInitializationException: The type initializer for 'SkiaSharp.SKObject' threw an exception.
 ---> System.InvalidOperationException: The version of the native libSkiaSharp library (88.1) is incompatible with this version of SkiaSharp. Supported versions of the native libSkiaSharp library are in the range [115.0, 116.0).
   at SkiaSharp.SkiaSharpVersion.CheckNativeLibraryCompatible(Version minSupported, Version current, Boolean throwIfIncompatible)
   at SkiaSharp.SkiaSharpVersion.CheckNativeLibraryCompatible(Boolean throwIfIncompatible)
   at SkiaSharp.SKObject..cctor()
   --- End of inner exception stack trace ---
   at SkiaSharp.SKObject..ctor(IntPtr handle, Boolean owns)
   at SkiaSharp.SKFontManager..ctor(IntPtr handle, Boolean owns)
   at SkiaSharp.SKFontManager.SKFontManagerStatic..ctor(IntPtr x)
   at SkiaSharp.SKFontManager..cctor()
   --- End of inner exception stack trace ---
   at SkiaSharp.SKFontManager.get_Default()
   at Avalonia.Skia.FontManagerImpl..ctor()
   at Avalonia.Skia.SkiaPlatform.Initialize(SkiaOptions options)
   at Avalonia.SkiaApplicationExtensions.<>c.<UseSkia>b__0_0()
   at Avalonia.AppBuilder.SetupUnsafe()
   at Avalonia.AppBuilder.Setup()
   at Avalonia.AppBuilder.SetupWithLifetime(IApplicationLifetime lifetime)
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime(AppBuilder builder, String[] args, Action`1 lifetimeBuilder)
   at SampleProject.Program.Main(String[] args) in /home/sobirjon/Documents/SampleProject/Program.cs:line 13
```

**Comment by jp2masa** [comment]
SkiaSharp 3.0 should work with latest nightly builds, but not stable releases. See #14510.

**Comment by timunie** [comment]
That's not a bug. 3.0 has breaking changes so will be available in 11.1 if everything goes right.
